src/main/java/wellnus/storage/Tokenizer.java

```diff
@@ -19,11 +18,12 @@
     /**
      * Converts the attributes of the given <code>Manager</code> into a String representation to be
      *     saved to storage.
-     * @param managerToTokenize Manager whose state we want to convert into a String representation
+     * @param dataObjects List of Objects which represent data we want to convert into a String representation
+     * @return ArrayList of Strings representing the data objects that we can write to storage
      * @throws TokenizerException If tokenizing fails and state cannot be converted into a valid String
      *     representation
      */
-    String tokenize(Manager managerToTokenize) throws TokenizerException;
+    ArrayList<String> tokenize(ArrayList<T> dataObjects) throws TokenizerException;
```

Comment:
Passing in an arraylist of data will make it easier to call the method and tokenize the data as compared to managers.